site/tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "moduleResolution": "node",
     "sourceMap": true,
     "esModuleInterop": true,
+    "skipLibCheck": true,
```

Comment:
why are we adding this ?

Reply:
https://github.com/multiformats/js-multiformats#typescript-support